.github/ISSUE_TEMPLATE/epic_release.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Epic Release
+about: Epic Release Checklist
+title: 'Epic Release: [epic name]'
+labels: epic-release-activities
+assignees: ''
+
+---
+
+The following is a checklist for new epics or features acceptance for Spartacus, based on [our Definition of Done](https://sap.github.io/spartacus-docs/definition-of-done/)
+
+## General
+
+- [ ] Epic has been tested on a production-like server using Spartacus installation script
+- [ ] Epic has been tested in all supported browsers and devices (Check list on DoD page)
+- [ ] Epic has sufficient end to end (cypress) test coverage
+- [ ] Schematics code deprecations have been provided and tested
+- [ ] Epic complies with [Security best practices](https://sap.github.io/spartacus-docs/security-best-practices/)
+- [ ] Epic complies with [Accessibility best practices](https://sap.github.io/spartacus-docs/a11y-best-practices/)
+- [ ] Epic CSS supports directionality (if required)
+- [ ] Epic code scans have been conducted (sonar, whitesource, checkmarx)
+
+## Audits/reviews
+
+- [ ] Architecture review of the epic (with @tobi-or-not-tobi)
+- [ ] Security and threat modeling audit/review with the Security team (main contact is @giancorderoortiz)
+- [ ] Accessibility audit/review (main contact is @developpeurweb)
+
+## New Library
+
+If your epic will be introduced or introduces a new library:
+
+- [ ] Library provides schematics to install the library
+- [ ] Library modules can be lazy loaded appropriately
+- [ ] Library has been tested for a customer using Spartacus installation script
+
+## Sample data
+
```

Comment:
- What if we have sample data for CI purposes that we don't want to share with customers (CPQ endpoints e.g.). For CPQ, our approach at the moment is to provide an extra script that will be deployed on top of commerce and sample data

Reply:
Noted. We assume that all of the sample data is public and free to be shared.